pmm/aws-staging-start.groovy

```diff
@@ -47,9 +47,17 @@ pipeline {
             defaultValue: '3.6',
             description: 'Percona Server for MongoDB version',
             name: 'MO_VERSION')
-        string(
+        text(
             defaultValue: '--addclient=ps,1',
-            description: 'Configure PMM Clients. ps - Percona Server for MySQL, pxc - Percona XtraDB Cluster, --with-proxysql (to be used with pxc only ex. --addclient=pxc,1 --with-proxysql) , ms - MySQL Community Server, md - MariaDB Server, mo - Percona Server for MongoDB, pgsql - Postgre SQL Server',
+            description: '''
+            Configure PMM Clients
+            ms - MySQL (ex. --addclient=ms,1), To Start with Slowlog use (--addclient=ms,1 --query-source=slowlog),
+            ps - Percona Server for MySQL (ex. --addclient=ps,1),
```

Comment:
Can we add an example here about Slow log. 
and for MariaDB too  ) 

thanks

Reply:
I have added for all type, slowlog for Percona server will be added this week, I will fix it then